runtime/jvmti/jvmtiHook.c

```diff
@@ -1589,7 +1589,7 @@ jvmtiHookGetEnv(J9HookInterface** hook, UDATA eventNum, void* eventData, void* u
 			|| (JVMTI_VERSION_9 == version)
 			|| (JVMTI_VERSION_11 == version)
 #if JAVA_SPEC_VERSION >= 15
-			|| (JVMTI_VERSION_15 == version)
+			|| (JVMTI_VERSION == version)
```

Comment:
This has the same problem as before - it won't accept versions prior to the current JDK level. This is where the bit check and mask and shift of major version is needed.

Reply:
Added the check `(J9_ARE_ALL_BITS_SET(version, JVMTI_VERSION_INTERFACE_JVMTI)  && (((version & JVMTI_VERSION_MASK_MAJOR) >> JVMTI_VERSION_SHIFT_MAJOR) <= JAVA_SPEC_VERSION))` here to accept earlier versions.
Also fold `JVMTI_VERSION_9` & `JVMTI_VERSION_11`.